src/ESLintLinter.php

```diff
@@ -84,10 +84,6 @@ public function getLinterConfigurationOptions() {
         'type' => 'optional string',
         'help' => pht('Specify environments. To specify multiple environments, separate them using commas. (https://eslint.org/docs/user-guide/command-line-interface#--env)'),
       ),
-      'eslint.fix' => array(
-        'type' => 'optional bool',
-        'help' => pht('Specify whether eslint should autofix issues. (https://eslint.org/docs/user-guide/command-line-interface#fixing-problems)'),
-      ),
```

Comment:
Dropping this flag is backwards-incompatible. Is there a nice way to preserve it for people who are relying on it? Perhaps the new behavior you've added could be conditional based on a boolean attribute that's set by this flag.